components/places/android/src/main/java/mozilla/appservices/places/PlacesConnection.kt

```diff
@@ -309,15 +310,7 @@ open class PlacesReaderConnection internal constructor(connHandle: Long, conn: U
     }
 
     override fun getTopFrecentSiteInfos(numItems: Int, frecencyThreshold: FrecencyThresholdOption): List<TopFrecentSiteInfo> {
-        val infoBuffer = rustCall { error ->
-            LibPlacesFFI.INSTANCE.places_get_top_frecent_site_infos(this.handle.get(), numItems, frecencyThreshold.value, error)
-        }
-        try {
-            val infos = MsgTypes.TopFrecentSiteInfos.parseFrom(infoBuffer.asCodedInputStream()!!)
-            return TopFrecentSiteInfo.fromMessage(infos)
-        } finally {
-            LibPlacesFFI.INSTANCE.places_destroy_bytebuffer(infoBuffer)
-        }
+        return this.conn.getTopFrecentSiteInfos(numItems, frecencyThreshold.value)
```

Comment:
surprised this wasn't using the `readQueryCounters.measure` dance! But agree with not adding it as part of this PR